index.js

```diff
@@ -13,10 +14,19 @@ module.exports = (options = { plugins: [] }) => ({
   setup: function (build) {
     const { rootDir = options.rootDir || process.cwd() } = options;
     const tmpDirPath = tmp.dirSync().name;
+
     build.onResolve(
       { filter: /.\.(css)$/, namespace: "file" },
       async (args) => {
-        const sourceFullPath = path.resolve(args.resolveDir, args.path);
+        let sourceFullPath;
+
+        // Use Node's module resolution algorithm for modules from node_modules
+        if (args.path.startsWith('.') || path.isAbsolute(args.path)) {
+          sourceFullPath = path.resolve(args.resolveDir, args.path);
+        } else {
+          sourceFullPath = resolve(args.path, { basedir: args.resolveDir });
```

Comment:
Is there not a way to do this without an external dependency? I'm not familiar with the `resolve` npm module

Reply:
Yes, I just pushed a change that should do it, I tested it with my build process and it works just as well as the `resolve` method.